### ISSUE TITLE
Issues configuring AWS Fargate with Prisma 1.7.1

### ISSUE THREAD

**Issue by vineetdpnd** [prismagraphql/prisma#2293]
We followed the steps to setup AWS Fargate mentioned on your blog :- 

https://blog.graph.cool/how-to-deploy-a-prisma-cluster-to-aws-fargate-using-docker-cloudformation-293aa8727b89

We were able to get the AWS side setup working totally fine.

We were earlier using a local docker on our EC2 , but now in order for us to scale and get production ready we would like to shift to AWS Fargate for future prisma deployments. Our Prisma Version was 1.6.* earlier on EC2 and it worked pretty well without any issues for our development requirements.

Since we upgraded to the latest prisma version 1.7.1 , and made changes to the endpoint in our prisma.yml as mentioned in the blog above & as per the new update 1.7.1 requirements, we are not able to execute the command prisma deploy. It results in the following error -> ther error is basically asking us to authenticate to prisma cloud for some reason which we are not able to figure out yet.

![image](https://user-images.githubusercontent.com/17204129/39333769-b6b470cc-49c9-11e8-88e9-c1cf9522437e.png)

**Comment by marktani** [comment]
Hey @vineetdpnd, thanks bringing this up, and the detailled description.

Are you able to reproduce this in a minimal setup? Can you pin point it to a specific step in the process?

**Comment by vineetdpnd** [comment]
Hey @marktani , we were able to finally deploy prisma with Prisma version 1.7.3 to AWS Fargate. Though we did it without using the Public & Private Key Authentication (since now the symmetric authentication approach has been implemented with the new Prisma version) which is suggested to be used with AWS Fargate Clusters as its an open endpoint. Currently however we have managed it via our AWS Security Groups. Thank you for your time.
